notification/src/config/init/ensure-adyen-webhook.js

```diff
@@ -0,0 +1,99 @@
+import fetch from 'node-fetch'
+import { serializeError } from 'serialize-error'
+import { getLogger } from '../../utils/logger.js'
+import config from '../config.js'
+
+const mainLogger = getLogger()
+
+async function ensureAdyenWebhook(adyenApiKey, webhookUrl, merchantId) {
+  try {
+    const logger = mainLogger.child({
+      adyen_merchant_id: merchantId,
+    })
+
+    const webhookConfig = {
+      type: 'standard',
+      url: webhookUrl,
+      active: 'true',
+      communicationFormat: 'json',
+      description: 'commercetools-adyen-integration notification webhook',
+    }
+
+    const getWebhookResponse = await fetch(
+      `https://management-test.adyen.com/v1/merchants/${merchantId}/webhooks`,
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'X-Api-Key': adyenApiKey,
+        },
+      }
+    )
+    const getWebhookResponseResponseJson = await getWebhookResponse.json()
```

Comment:
```suggestion
    const getWebhookResponseJson = await getWebhookResponse.json()
```
It looks better this way.